server/controllers/stock/reports/stock/adjustment_receipt.js

```diff
@@ -13,20 +13,27 @@ const {
 async function stockAdjustmentReceipt(documentUuid, session, options) {
   const optionReport = _.extend(options, { filename : 'STOCK.REPORTS.ADJUSTMENT' });
 
+  const FLUX_TYPE = [
+    Stock.flux.FROM_ADJUSTMENT,
+    Stock.flux.TO_ADJUSTMENT,
+    Stock.flux.INVENTORY_RESET,
+    Stock.flux.INVENTORY_ADJUSTMENT,
+  ];
+
   // set up the report with report manager
   const report = new ReportManager(STOCK_ADJUSTMENT_TEMPLATE, session, optionReport);
   const sql = `
     SELECT i.code, i.text, BUID(m.document_uuid) AS document_uuid, m.is_exit,
       m.quantity, m.unit_cost, (m.quantity * m.unit_cost) AS total , m.date, m.description,
       u.display_name AS user_display_name, dm.text AS document_reference,
-      l.label, l.expiration_date, d.text AS depot_name
+      l.label, l.expiration_date, d.text AS depot_name, m.flux_id
     FROM stock_movement m
     JOIN document_map dm ON dm.uuid = m.document_uuid
     JOIN lot l ON l.uuid = m.lot_uuid
     JOIN inventory i ON i.uuid = l.inventory_uuid
     JOIN depot d ON d.uuid = m.depot_uuid
     JOIN user u ON u.id = m.user_id
-    WHERE m.flux_id IN (${Stock.flux.FROM_ADJUSTMENT}, ${Stock.flux.TO_ADJUSTMENT}) AND m.document_uuid = ?
+    WHERE m.flux_id IN (${FLUX_TYPE}) AND m.document_uuid = ?
```

Comment:
Just for clarify, can you make this into `FLUX_TYPE.join(',')`?  I know that be default, it will call `Array.prototype.toString()` and that the `toString()` method joins the array with a comma, but someone reading the code in the future may not know that.